packages/flutter/lib/src/widgets/text.dart

```diff
@@ -99,6 +99,10 @@ class DefaultTextStyle extends InheritedTheme {
   /// [DefaultTextStyle] using the [DefaultTextStyle.new] constructor directly.
   /// See the source below for an example of how to do this (since that's
   /// essentially what this constructor does).
+  ///
+  /// This constructor replaces the [textHeightBehavior] object without
+  /// attempting a merge. If you want to merge with the [textHeightBehavior],
```

Comment:
```suggestion
  /// If a [textHeightBehavior] is provided, the existing configuration will be
  /// replaced compeletely. To retain part of the original [textHeightBehavior],
```

When I first read "This constructor replaces the `textHeightBehavior` object", I was thinking it would be replaced even if no argument was passed.

<br>

And 1 other thing from the [style guide](https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#use-the-passive-voice-recommend-do-not-require-never-say-things-are-simple):

> ### Use the passive voice; recommend, do not require; never say things are simple
> Never use "you" or "we".